phpstan-dbal3.neon

```diff
@@ -0,0 +1,14 @@
+includes:
+    - phpstan.neon
+
+parameters:
+    ignoreErrors:
+        # The class was added in PHP 8.1
+        - '/^Attribute class ReturnTypeWillChange does not exist.$/'
+
+        # https://github.com/doctrine/collections/pull/282
+        - '/Variable \$offset in isset\(\) always exists and is not nullable\./'
```

Comment:
Shouldn't the 2 errors above be in `phpstan.neon`?

Reply:
It looks like it might require a bit more work - as the ignored paths are merged from all files. May it would make sense to have 2 separate baselines and configs for dbal2 and dbal3?

Reply:
Maybe yes, but I'd rather do that when absolutely necessary.

Reply:
@greg0ire - splitted a little bit more. Common params moved to separate file.